include/osmium/osm/timestamp.hpp

```diff
@@ -238,11 +238,13 @@ namespace osmium {
          * Return the timestamp as string in ISO date/time
          * ("yyyy-mm-ddThh:mm:ssZ") format. If the timestamp is invalid, an
          * empty string will be returned.
+         *
+         * @param output_zero output the string even if it is 1970-01-01T00:00:00Z
          */
-        std::string to_iso() const {
+        std::string to_iso(const bool output_zero = false) const {
             std::string s;
 
-            if (m_timestamp != 0) {
+            if (m_timestamp != 0 || output_zero) {
```

Comment:
After looking again at this code an my changes to it, I wonder if it wouldn't be better to return `1970-01-01T00:00:00Z` always if the timestamp is 0. We need something in the XML `timestamp` attribute. Or should OPL files don't have timestamps if they are 0.